--- origsrc/libj2k-0.0.9/src/Makefile.am	2004-03-01 18:48:31.000000000 -0600
+++ src/libj2k-0.0.9/src/Makefile.am	2007-06-20 17:39:46.112307300 -0500
@@ -1,4 +1,5 @@
 lib_LTLIBRARIES = libj2k.la
 libj2k_la_SOURCES = j2k.c j2k.h tcd.c tcd.h t1.c t1.h t2.c t2.h tgt.c tgt.h mqc.c mqc.h bio.c bio.h int.c int.h mct.c mct.h dwt.c dwt.h cio.c cio.h fix.c fix.h pi.c pi.h log.c log.h
+libj2k_la_LDFLAGS = -no-undefined
 include_HEADERS = j2k.h
 AM_CFLAGS = -O3 -funroll-loops
